data/autoyast_containers.xml.ep

```diff
@@ -47,6 +47,14 @@
     <packages config:type="list">
       <package>iputils</package>
       <package>acl</package>
+      <package>docker</package>
```

Comment:
this currently used only for SLE ? just in case you forgot this won't work with openSUSE

Reply:
This profile is used for SLE only. Maybe the filename should be more specific...